### PR TITLE
Env app describe and ls changes 

### PR DESCRIPTION
**Pull request by anna-cross**
## Description of change

We are no longer pulling type and provider info for Envs on Apps. Making changes according to that. 

Fixes <https://github.com/meroxa/cli/issues/622>

## Type of change

- [x]  Refactor

## How was this tested?

- [x]  Unit Tests
- [x]  Tested in staging
- [ ]  Tested in minikube

## DEMO 

# APP DESCRIBE WITH PRIVATE ENV AND  IN COMMON 

image

# APP LS WITH PRIVATE ENV AND IN COMMON 

image

# APP LS JUST COMMON 

image

# APP LS JUST PRIVATE ENV 

image

### CODE REVIEW

File: utils/display/apps.go [hunk 1]
```diff
@@ -37,10 +33,13 @@ func AppsTable(apps []*meroxa.Application, hideHeaders bool) string {
...
 		env, err := app.Environment.GetNameOrUUID()
 
-		if err == nil && itShouldEnvInfo(app.Environment) {
+		if err == nil {
 			withEnvironment = true
 			r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: env})
 		}
+		if withEnvironment == true && err != nil {
+			r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: "common"})
+		}
```
Comment (dianadoherty) [inline utils/display/apps.go:42]: Logic here seems a little weird
I feel like we can also check if env is nil with `app.Environment != nil` for whether the app is on a common env vs not

Comment (anna-cross) [reply utils/display/apps.go:42]: So, in this I'm trying to cover a case  when we do add an Environment column on table for an app with a private env, but there is also an app without environment (aka common). 
If app is common, app.Environment will be nil , but we would still want to add a text as the simple table starts panicking when Environment column exists but we pass it no value in a row 🤔

Comment (dianadoherty) [reply utils/display/apps.go:42]: Yeah, but the only time `withEnvironment` is set to true if `err == nil`, so the second if statement will never work

Comment (anna-cross) [reply utils/display/apps.go]: Looking at it again, ye it makes no sense. Made some changes. We are now adding Env column no matter what and just filling it with common if an app doesn't have a private app

File: utils/display/apps.go [hunk 2]
```diff
@@ -35,12 +29,15 @@ func AppsTable(apps []*meroxa.Application, hideHeaders bool) string {
...
-		env, err := app.Environment.GetNameOrUUID()
-
-		if err == nil && itShouldEnvInfo(app.Environment) {
-			withEnvironment = true
-			r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: env})
+		if app.Environment != nil {
+			env, err := app.Environment.GetNameOrUUID()
+			if err == nil {
+				r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: env})
+			}
```
Comment (dianadoherty) [inline utils/display/apps.go:36]: The error case isn't getting handled here. Instead I recommend being explicit about whether you want to include the name or the uuid here. 

Here's an example for resources https://github.com/meroxa/cli/blob/2a0ff49ec84721823307244176272a18b0bd5437/utils/display/resources.go#L53-L65

**Review by dianadoherty — APPROVED** [review]

**Pull request merged**